charts/sampling/templates/_otelcol_processor_transform.river.txt

```diff
@@ -0,0 +1,69 @@
+{{- define "deployment.processor.transform.add_resource_attributes" -}}
+otelcol.processor.transform "add_resource_attributes_as_metric_attributes" {
```

Comment:
Ditto, is this for incoming metrics or for metrics produced internally?
I'm not sure to catch the role for a "sampling box".